types/k6/http.d.ts

```diff
@@ -242,7 +242,7 @@ export interface Params {
     tags?: { [name: string]: string };
 
     /** Request timeout. */
-    timeout?: number;
+    timeout?: string;
```

Comment:
Hmm this would now work with both numbers _and_ strings - we just formalize that numbers everywhere mean milliseconds.

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/51236/commits/ea80594417252a05c9336b1a4eaee777fb13a711